basis-universal/src/transcoding/enums.rs

```diff
@@ -188,7 +188,10 @@ impl TranscoderTextureFormat {
 
     /// Returns true if the transcoder texture type is a compressed format.
     pub fn is_compressed(self) -> bool {
-        unsafe { !sys::basis_transcoder_format_is_uncompressed(self.into()) }
```

Comment:
I actually don't think it's good to rewrite the C code into rust. If upstream implementation changes, we would have no way of knowing and it would just be silently wrong.

Reply:
That's fair, I can have a go at writing bindings for these.